bigtable/google/cloud/bigtable_v2/gapic/bigtable_client.py

```diff
@@ -97,6 +97,10 @@ def __init__(self,
                 self.SERVICE_ADDRESS,
                 credentials=credentials,
                 scopes=self._DEFAULT_SCOPES,
+                options={
```

Comment:
Hrm, I'm a little worried about fragility here. Do we make users construct this directly or do we have a wrapper class around this?

Reply:
The wrapper class is `google.cloud.bigtable.client.Client`, which constructs / holds an instance of this class as its `table_data_client` property.

Reply:
wanna move that there? (like we do in pubsub)?

Reply:
This *is* the equivalent location for `pubsub_v1/publisher/client.py` / `pubsub_v1/subscriber/client.py`:  there is no wrapper client class there.